test-clients/src/main/java/com/hedera/services/bdd/suites/perf/file/MixedFileOpsLoadTest.java

```diff
@@ -0,0 +1,115 @@
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import com.hedera.services.bdd.spec.HapiApiSpec;
+import com.hedera.services.bdd.spec.HapiSpecOperation;
+import com.hedera.services.bdd.spec.transactions.TxnUtils;
+import com.hedera.services.bdd.spec.utilops.LoadTest;
+import com.hedera.services.bdd.suites.perf.PerfTestLoadSettings;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.function.Supplier;
+
+import static com.hedera.services.bdd.spec.HapiApiSpec.defaultHapiSpec;
+import static com.hedera.services.bdd.spec.queries.QueryVerbs.getFileInfo;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.fileAppend;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.fileCreate;
+import static com.hedera.services.bdd.spec.transactions.TxnVerbs.fileUpdate;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.logIt;
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.withOpContext;
+
+/**
+ * Run mixed operations including FileCreate, FileAppend, FileUpdate
+ */
+public class MixedFileOpsLoadTest extends LoadTest {
+	private static final Logger log = LogManager.getLogger(MixedFileOpsLoadTest.class);
+
+	public static void main(String... args) {
+		parseArgs(args);
+
+		MixedFileOpsLoadTest suite = new MixedFileOpsLoadTest();
+		suite.setReportStats(true);
+		suite.runSuiteSync();
+	}
+
+	@Override
+	protected List<HapiApiSpec> getSpecsInSuite() {
+		return List.of(
+				runMixedFileOps()
+		);
+	}
+
+	@Override
+	public boolean hasInterestingStats() {
+		return true;
+	}
+
+	protected HapiApiSpec runMixedFileOps() {
+		PerfTestLoadSettings settings = new PerfTestLoadSettings();
+		final AtomicInteger submittedSoFar = new AtomicInteger(0);
+		final byte[] newContents = TxnUtils.randomUtf8Bytes(TxnUtils.BYTES_4K);
+		String initialContent = "The initial contents!";
+		String targetFile = "targetFile";
+
+		Supplier<HapiSpecOperation[]> mixedFileOpsBurst = () -> new HapiSpecOperation[] {
+				fileCreate(targetFile + submittedSoFar.getAndIncrement()).contents(initialContent),
+				fileUpdate(targetFile)
+						.fee(ONE_HUNDRED_HBARS)
+						.contents(newContents)
+						.noLogging()
+						.payingWith(GENESIS)
+						.hasAnyPrecheck()
+						.deferStatusResolution(),
+				fileAppend(targetFile)
+						.content("dummy")
+						.hasAnyPrecheck()
+						.payingWith(GENESIS)
+						.fee(ONE_HUNDRED_HBARS)
+						.logging()
+						.deferStatusResolution()
```

Comment:
nit: These two operations will operate on the same file object.  Is this intended to be a hotspot perf testing?

Reply:
@ljianghedera  This is not intended to be for performance testing though it extends LoadTest. This test is supposed to submit  file operations and modify few files during reconnect. As the files are blobs I believe they will create new blobs if modified.

Reply:
Got it.